src/routes/Manual.svelte

```diff
@@ -129,9 +129,7 @@ function setTimingBlock(config) {
 
 // MARK: Device Input
 
-$: activeHexagon = -1;
-$: mouseDown = false;
-$: pendingTimeout = false;
+let activeHexagon = -1;
 
 let block0_31 = [0,0,0,0];
```

Comment:
If we could pull out the block math and command building, this file could start to be really clean. Pulling this apart further would be overkill. I think this is just about the right amount of complexity until there are further needs.